doc/index.rst

```diff
@@ -232,7 +238,12 @@ Gitflow Workflow
 
 See `Gitflow Workflow <https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow>`_ for the detailed description of the workflow and process.
 
-**Master branch**
+Our team uses a modified gitflow workflow.
+The main difference is that most of the repositories do not use the Release branch and instead choose to merge to main from develop directly.
+This is because most of our repositories only have one or two developers working on them at a time and releases are usually coordinated so that no one can accidently commit changes to a release.
+Some of our larger repositories such as ts_xml and ts_sal which have a larger number of developers working on them, use the Release branch.
+
+**Main branch**
```

Comment:
Make these subsections instead of bold text (here and the other occurrences as well).